--- pastebin-0.60/lib/config/default.conf.php	2012-03-10 14:00:39.605878805 +0200
+++ pastebin/lib/config/default.conf.php	2012-03-10 14:06:33.734096378 +0200
@@ -51,25 +51,25 @@
 /**
 * Email address feedback should be sent to
 */
-$CONF['feedback_to']='lordelph@gmail.com';
+$CONF['feedback_to']='nobody@example.org';
 
 /**
 * Apparent sender address for feedback email
 */
-$CONF['feedback_sender']='pastebin.com <no-reply@pastebin.com>';
+$CONF['feedback_sender']='pastebin <nobody@example.org>';
 
 /**
 * database type - can be file or mysql
 */
-$CONF['dbsystem']='mysql';
+$CONF['dbsystem']='file';
 
 /**
 * db credentials
 */
 $CONF['dbhost']='localhost';
 $CONF['dbname']='pastebin';
-$CONF['dbuser']='pastebin';
-$CONF['dbpass']='banjo';
+$CONF['dbuser']='mysql';
+$CONF['dbpass']='';
 
 /**
  * format of urls to pastebin entries - %d is the placeholder for
@@ -85,21 +85,20 @@
  * 3. if not using mod_rewrite, you'll need something more like this:
  *    $CONF['url_format']="/pastebin.php?show=%s";
  */
-$CONF['url_format']='/%s';
-
+$CONF['url_format']='/pastebin/?show=%s';
 
 
 /**
 * default expiry time d (day) m (month) or f (forever)
 */
-$CONF['default_expiry']='m';
+$CONF['default_expiry']='f';
 
 /**
 * this is the path to the script - you may want to
 * to use / for even shorter urls if the main script
 * is renamed to index.php
 */
-$CONF['this_script']='/pastebin.php';
+$CONF['this_script']='pastebin.php';
 
 /**
 * what's the maximum number of posts we want to keep?
@@ -123,8 +122,9 @@
 
 /**
 * Google Adsense, clear this to remove ads. 
+* $CONF['google_ad_client']='pub-3281640380846080';
 */
-$CONF['google_ad_client']='pub-3281640380846080';
+$CONF['google_ad_client']='';
 
 /**
 * maintainer mode enables some code used to aid translation - unless you
