--- ./public_html/./pastebin.php~	2007-07-10 17:57:12.000000000 +0300
+++ ./public_html/./pastebin.php	2012-03-10 13:58:09.986257287 +0200
@@ -27,7 +27,7 @@
 // includes
 //
 require_once('pastebin/config.inc.php');
-require_once('geshi/geshi.php');
+require_once('geshi.php');
 require_once('pastebin/diff.class.php');
 require_once('pastebin/pastebin.class.php');
 
--- ./public_html/./translate.php~	2007-07-10 18:45:42.000000000 +0300
+++ ./public_html/./translate.php	2012-03-10 13:58:17.486572502 +0200
@@ -27,7 +27,7 @@
 // includes
 //
 require_once('pastebin/config.inc.php');
-require_once('geshi/geshi.php');
+require_once('geshi.php');
 require_once('pastebin/diff.class.php');
 require_once('pastebin/pastebin.class.php');
 
@@ -144,4 +144,4 @@
 
 
 
-		
\ No newline at end of file
+		
